query/src/test/java/org/infinispan/query/backend/MultipleEntitiesTest.java

```diff
@@ -70,15 +56,6 @@ public void testIndexAndQuery() {
       CacheQuery<?> queryDeb = searchManager.getQuery("FROM " + Debenture.class.getName());
       assertEquals(queryDeb.getResultSize(), 1);
    }
-
-   private void assertEfficientIndexingUsed(SearchIntegrator searchIntegrator, Class<?> clazz) {
-      DirectoryBasedIndexManager im = (DirectoryBasedIndexManager) searchIntegrator.getIndexBindings().get(clazz)
-            .getIndexManagerSelector().all().iterator().next();
-      WorkspaceHolder workspaceHolder = im.getWorkspaceHolder();
-      LuceneBackendResources indexResources = workspaceHolder.getIndexResources();
-      IndexWorkVisitor<Void, LuceneWorkExecutor> visitor = indexResources.getWorkVisitor();
-      assertTrue(TestingUtil.extractField(visitor, "updateExecutor") instanceof ByTermUpdateWorkExecutor);
```

Comment:
This test is pretty ancient, but it was testing if the index operations were done using   [ByTermUpdateWorkExecutor](https://docs.jboss.org/hibernate/search/5.4/api/org/hibernate/search/backend/impl/lucene/works/ByTermUpdateWorkExecutor.html) instead of the  [UpdateWorkExecutor](https://docs.jboss.org/hibernate/search/5.4/api/org/hibernate/search/backend/impl/lucene/works/UpdateWorkExecutor.html). The motivation was [ISPN-5103](ISPN-5103)

We have a knob in the infinispan integration to activate this in ```SearchableCacheConfiguration.isDeleteByTermEnforced()```. How does this translate to Hibernate Search 6.0?  Is it possible to instruct it to do delete by terms in the lucene backend?

Reply:
Hibernate Search 6 does not allow storing multiple entity types in the same index. Each entity type has its own index, even protobuf entity types. Thus all deletes are deletes by term (... unless you're using multi-tenancy, but that's not relevant here).
So maybe this test is no longer relevant?

Reply:
Ok, cool, this can be deleted then!